.eslintrc

```diff
@@ -10,15 +10,20 @@
     "react/prefer-es6-class": 0,
     "react/prefer-stateless-function": 0,
     "react/no-multi-comp": 0,
-    "array-callback-return": "off", 
+    "array-callback-return": "off",
     "arrow-body-style": ["warn", "as-needed"],
-    "comma-dangle": ["warn", "never"],
+    "comma-dangle": ["warn", "always"],
```

Comment:
I don't think we should require trailing comments. Parrish made a good point when we turned this off in #3328 that no other language allows this. I set it as a warning because it had been default enabled with the airbnb ruleset and some folks had been using them. The warning was so folks know to stop using it.